### PR TITLE
fix several personal settings problems

### PR DESCRIPTION
**Pull request by szaimen**
Address part of https://github.com/nextcloud/server/issues/36965

| Before | After Mobile | After Desktop |
|---|---|---|
| image | image | image |
| image | image | image |
| image | image | image |

### CODE REVIEW

File: apps/dav/src/views/Availability.vue [hunk 1]
```diff
@@ -155,51 +157,45 @@ export default {
 }
 .time-zone {
 	padding: 32px 12px 12px 0;
+    display: flex;
+    flex-wrap: wrap;
+
+	&__heading {
+		margin-bottom: calc(var(--default-grid-baseline) * 2);
```
Comment (szaimen) [inline apps/dav/src/views/Availability.vue:164]: Rm

File: apps/dav/src/views/Availability.vue [hunk 2]
```diff
@@ -126,20 +126,22 @@ export default {
...
-	white-space: nowrap;
+	white-space: normal;
 }
-.availability-slot {
+::v-deep .availability-slot {
 	display: flex;
 	flex-direction: row;
 	align-items: center;
+	flex-wrap: wrap;
+	justify-content: center;
```
Comment (szaimen) [inline apps/dav/src/views/Availability.vue:142]: Rm

File: apps/dav/src/views/Availability.vue [hunk 3]
```diff
@@ -126,20 +126,21 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.availability-day {
+::v-deep .availability-day {
```
Comment (Pytal) [inline apps/dav/src/views/Availability.vue:129]:
```suggestion
:deep(.availability-day) {
```

File: apps/dav/src/views/Availability.vue [hunk 4]
```diff
@@ -126,20 +126,21 @@ export default {
...
 	padding: 0 10px 0 10px;
 	position: absolute;
 }
-.availability-slots {
+::v-deep .availability-slots {
```
Comment (Pytal) [inline apps/dav/src/views/Availability.vue:133]:
```suggestion
:deep(.availability-slots) {
```

File: apps/dav/src/views/Availability.vue [hunk 5]
```diff
@@ -126,20 +126,21 @@ export default {
...
 	display: flex;
-	white-space: nowrap;
+	white-space: normal;
 }
-.availability-slot {
+::v-deep .availability-slot {
```
Comment (Pytal) [inline apps/dav/src/views/Availability.vue:137]:
```suggestion
:deep(.availability-slot) {
```

File: apps/dav/src/views/Availability.vue [hunk 6]
```diff
@@ -126,20 +126,21 @@ export default {
...
 	display: flex;
 	flex-direction: row;
 	align-items: center;
+	flex-wrap: wrap;
 }
-.availability-slot-group {
+::v-deep .availability-slot-group {
```
Comment (Pytal) [inline apps/dav/src/views/Availability.vue:143]:
```suggestion
:deep(.availability-slot-group) {
```

File: apps/dav/src/views/Availability.vue [hunk 7]
```diff
@@ -155,51 +156,44 @@ export default {
...
 	grid-row-gap: 6px;
-	grid-template-columns: min-content min-content min-content;
+	grid-template-columns: min-content auto min-content;
+	max-width: 500px;
 }
 .button {
 	align-self: flex-end;
 }
-.label-weekday {
+::v-deep .label-weekday {
```
Comment (Pytal) [inline apps/dav/src/views/Availability.vue:178]:
```suggestion
:deep(.label-weekday) {
```

File: apps/dav/src/views/Availability.vue [hunk 8]
```diff
@@ -155,51 +156,44 @@ export default {
...
 
-.add-another {
-	background-color: transparent;
-	border: none;
-	opacity: .5;
-	display: inline-flex;
-	padding: 0;
-	margin: 0;
-	margin-bottom: 3px;
+::v-deep .delete-slot {
```
Comment (Pytal) [inline apps/dav/src/views/Availability.vue:185]:
```suggestion
:deep(.delete-slot) {
```

File: apps/dav/src/views/Availability.vue [hunk 9]
```diff
@@ -155,51 +156,44 @@ export default {
...
+	padding-bottom: unset;
+}
 
-	&:hover {
-		opacity: 1;
-	}
+::v-deep .add-another {
```
Comment (Pytal) [inline apps/dav/src/views/Availability.vue:189]:
```suggestion
:deep(.add-another) {
```

**Review by Pytal — APPROVED** [review]
Other than the syntax update

**Review by AndyScherzinger — APPROVED** [review]
Only visually checked, LGTM 💙

**Comment by szaimen** [comment]
CI failure unrelated

**Pull request merged**